### ISSUE TITLE
cy.route2 stubbed calls remain pending ( with loading icon ) even if they are completed

### ISSUE THREAD

**Issue by jquintozamora** [cypress-io/cypress#9072]
### Current behavior
When stubbing using cy.route2 then the UI has some inconsistences. For example, the stubbed call keep loading even if the request has been completed.

![image](https://user-images.githubusercontent.com/12530031/97984718-0be62500-1dcf-11eb-845f-0288a5fa76d6.png)

### Desired behavior
UI is updated like the other requests:

![image](https://user-images.githubusercontent.com/12530031/97984919-5bc4ec00-1dcf-11eb-9c54-7d0d9f0647ee.png)

So we are able to pin them.

### Versions

version 5.5.0

**Comment by BrettWalker05** [comment]
Hi @jquintozamora , I have recently started using the cy.route2 since our upgrade to cypress 5.5.0. I also noticed this pending state for any intercepted req. I also noticed that with these pending req my test commands are not being fully executed and the tests are still marked as passed, have you encountered this problem? See a copy of my test runner where the test just stops executing and it moves onto the next test. When I remove the cy.route2 intercept the tests execute fine.

![Screenshot 2020-11-03 at 17 09 14](https://user-images.githubusercontent.com/46792277/98018139-179a1180-1df8-11eb-83cf-335c2d23fa40.png)
